CMakeLists.txt

```
@@ -1,50 +1,106 @@
-cmake_minimum_required(VERSION 3.10)
-project(AppwriteSDK CXX)
-
-set(CMAKE_CXX_STANDARD 11)
-
-include_directories(${CMAKE_BINARY_DIR}/conan/include)
-link_directories(${CMAKE_BINARY_DIR}/conan/lib)
-
-find_package(CURL REQUIRED)
-
-set(SRCS
-    src/Appwrite.cpp
-    src/services/Account.cpp
-    src/services/Query.cpp
-    src/services/Databases.cpp
-    src/services/Messaging.cpp
-    src/services/Storage.cpp
-    src/services/Health.cpp
-    src/Utils.cpp
-    src/Validator.cpp
-)
+cmake_minimum_required(VERSION 3.15)
+project(AppwriteSDK VERSION 1.0.0 LANGUAGES CXX)
 
-add_library(AppwriteSDK STATIC ${SRCS})
+# Set C++ standard
+set(CMAKE_CXX_STANDARD 17)
```

```suggestion
set(CMAKE_CXX_STANDARD 11)
```

This should be 17 i guess, keeping the same as the current standard.

---

i have set the standard set as 17 only is there an issue?

---

but afaik libcurl was failing its build in version 17